### DISCUSSION TITLE
[Windows] Why Title not be shown?

### DISCUSSION THREAD

**Discussion by emako** [AvaloniaUI/Avalonia#15402] [Q&A]
![image](https://github.com/AvaloniaUI/Avalonia/assets/24737061/4effc9aa-d1ae-47b9-b39a-1d9646efd161)

```xaml
<!-- MainWindow.axaml -->
Title="LyricStudio"
```

**Comment by rabbitism** [comment]
If you hide system chrome, title is also hidden. You need to place it in your view by yourself.

**Comment by emako** [comment]
![image](https://github.com/AvaloniaUI/Avalonia/assets/24737061/f326b7f5-6dbd-4422-9ebf-333c43ea9164)

But I didn't hide system chrome like `SystemDecorations="Full"`.
 
```xaml

 <Window x:Class="LyricStudio.Views.MainWindow"
                  xmlns="https://github.com/avaloniaui"
                  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
                  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
                  Title="LyricStudio"
                  Width="1000"
                  Height="600"
                  d:DesignHeight="450"
                  d:DesignWidth="800"
                  Icon="/Assets/Images/Favicon.ico"
                  SystemDecorations="Full"
                  mc:Ignorable="d"/>
```

**Reply by timunie** [reply]
please file a minimal sample App to double-check.

**Comment by emako** [comment]
May here is a better method to custmize title bar.
https://github.com/AvaloniaUI/Avalonia/discussions/12090

**Comment by emako** [comment]
I Found the reason:
       
 UxTheme.SetWindowThemeAttribute(
            hWnd,
            UxTheme.WINDOWTHEMEATTRIBUTETYPE.WTA_NONCLIENT,
            wtaOptions,
            (uint)Marshal.SizeOf(typeof(UxTheme.WTA_OPTIONS))
        );
